packages/core/src/js/feedback/FeedbackFormScreen.tsx

```diff
@@ -0,0 +1,102 @@
+import { captureFeedback } from '@sentry/core';
+import type { SendFeedbackParams } from '@sentry/types';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import { Alert, Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import defaultStyles from './FeedbackFormScreen.styles';
+import type { FeedbackFormScreenProps, FeedbackFormScreenState } from './FeedbackFormScreen.types';
+
+/**
+ * Implements a feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
```

Comment:
Let's add beta label, it's not a ricky change so we can release this without beta of the whole SDK, but at first I would label it as beta to collect some initial feedback and have a bit of room for small changes.

Reply:
Great idea! updated with 1bc1e4c5cc33a1080f0e0768e8ff221351107fde